src/west/manifest.py

```diff
@@ -1062,6 +1052,10 @@ def posixpath(self):
             self._posixpath = PurePath(self.abspath).as_posix()
         return self._posixpath
 
+    @property
```

Comment:
I didn't realize property decorators were also not supported in 3.4

Reply:
They are, it's just so we can use {name_and_path} from f-strings.